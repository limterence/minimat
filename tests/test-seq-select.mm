/* check-seq-select -- sequence index out of bounds*/
int main() {
  sequence a;
  a = new sequence(2);
  printseq(a[2]);        /* cannot reference sequence out of bounds */
}

