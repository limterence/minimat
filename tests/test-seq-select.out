sequence index out of bounds.  Exiting...
